dbscript/dbscript-1.sql

```diff
@@ -58,14 +58,14 @@ create table `hs_hr_payperiod` (
 ) engine=innodb default charset=utf8;
 
 create table `hs_hr_emp_basicsalary` (
-  `id` INT AUTO_INCREMENT, 

```

Comment:
file contains formatting changes and it's difficult to identify the actual changes to this file.
As the schema is changed we may have to handle this in upgrader as well.

Reply:
this is just a format changing when save the file